### PR TITLE
MSSQL ODBC arm64 support

### LINKED ISSUE #6659: can not retrieving schema from MSSQL Server
### Issue Summary

I get time out error when  I want to retrieve the database schema to execute a query and get this error:
```
{
    "job": {
        "id": "793ec547-981f-4685-850a-f3636881742f",
        "updated_at": 0,
        "status": 4,
        "error": "b'DB-Lib error message 20009, severity 9:\\nUnable to connect: Adaptive Server is unavailable or does not exist (172.25.143.154)\\nNet-Lib error during Connection timed out (110)\\n'",
        "result": null,
        "query_result_id": null
    }
}
``` 
It is worth noting that the connection to the database is also established after a long time. Also, in the firewall and database, access through IP and port is open and the Database is accessible from MSS Management Studio.

### Steps to Reproduce

1. Install the Redash from wiki instruction [Link](https://github.com/getredash/redash/wiki/Local-development-setup)
2. Add MSSQL Database and  test connection then try to add and execute query
3. Get above error

### Technical details:

* Redash Version:  23.12.0-dev (dev)
* Browser/OS: Windows 10 - WSL - Chrome
* How did you install Redash: Install the Redash from wiki instruction [Link](https://github.com/getredash/redash/wiki/Local-development-setup)

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
## What type of PR is this? 
Added MSSQL ODBC arm64 support

- [ ] Refactor
- [x] Feature
- [x] Bug Fix
- [ ] New Query Runner (Data Source) 
- [ ] New Alert Destination
- [ ] Other

## How is this tested?

- [x] Unit tests (pytest, jest)
- [x] E2E Tests (Cypress)
- [x] Manually
- [ ] N/A

Tested using docker compose on linux/arm64. added MSSQL image to compose and configured datasource on Redash

```
mssql:
    environment:
      ACCEPT_EULA: Y
      MSSQL_SA_PASSWORD: Testtest1!
    image: mcr.microsoft.com/azure-sql-edge
```
Screenshot 2024-04-24 at 12 13 48
Screenshot 2024-04-24 at 12 14 43

### CODE REVIEW

**Comment by justinclift** [comment]
Ahhh, cool.  The lack of ARM64 support in our MS SQL data sources has been a problem for a while.

It'll be good to get this fixed. :smile:

File: redash/query_runner/mssql_odbc.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def configuration_schema(cls):
                 "verify_ssl": {
                     "type": "boolean",
                     "title": "Verify SSL certificate",
-                    "default": True,
+                    "default": False,
```
Comment (justinclift) [inline redash/query_runner/mssql_odbc.py:51]: Changing defaults can be a problem if we're not careful.

I *think* it should be ok in this instance though. :smile:

**Comment by justinclift** [comment]
@AndrewChubatiuk How has this been tested?  There's no info on that in the PR description bit. :wink:

**Comment by AndrewChubatiuk** [comment]
> @AndrewChubatiuk How has this been tested? There's no info on that in the PR description bit. 😉

I've added to compose mssql container, created datasource and queries it's data then
```
mssql:
    environment:
      ACCEPT_EULA: Y
      MSSQL_SA_PASSWORD: Testtest1!
    image: mcr.microsoft.com/azure-sql-edge
```

**Comment by AndrewChubatiuk** [comment]
> @AndrewChubatiuk How has this been tested? There's no info on that in the PR description bit. 😉

@justinclift I've tested it on linux/arm64. Added mssql container to compose, created datasource and queried it's data then

```mssql:
    environment:
      ACCEPT_EULA: Y
      MSSQL_SA_PASSWORD: Testtest1!
    image: mcr.microsoft.com/azure-sql-edge
    ```

File: redash/query_runner/mssql_odbc.py (hunk 1 above)
Comment (AndrewChubatiuk) [reply redash/query_runner/mssql_odbc.py:51]: there's a problem in ui, which doesn't set toggle to enabled state by default, when default True value is set in a schema

**Comment by AndrewChubatiuk** [comment]
> @AndrewChubatiuk How has this been tested? There's no info on that in the PR description bit. 😉

@justinclift updated description

File: redash/query_runner/mssql_odbc.py (hunk 1 above)
Comment (justinclift) [reply redash/query_runner/mssql_odbc.py:51]: k, that's something we should probably look at later on then. :smile:

**Review by justinclift — APPROVED** [review]
Looks good to me.  Thanks @AndrewChubatiuk. :smile:

**Pull request merged**